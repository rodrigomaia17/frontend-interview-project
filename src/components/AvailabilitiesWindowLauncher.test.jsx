import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Launcher from './AvailabilitiesWindowLauncher.jsx';

Enzyme.configure({ adapter: new Adapter() });

it('renders correctly closed', () => {
  const c = shallow(<Launcher />);
  const tree = toJson(c)
  expect(tree).toMatchSnapshot();
});

it('renders correctly closed', () => {
  const c = shallow(<Launcher />);
  c.setState({isOpen: true})
  const tree = toJson(c)
  expect(tree).toMatchSnapshot();
});
