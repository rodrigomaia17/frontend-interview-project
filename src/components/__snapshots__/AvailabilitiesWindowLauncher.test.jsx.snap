// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly closed 1`] = `
<styled.div
  onClick={[Function]}
/>
`;

exports[`renders correctly closed 2`] = `
<AvailabilitiesWindow
  onClose={[Function]}
/>
`;
